packaging/installer/install-required-packages.sh

```diff
@@ -129,1282 +128,1220 @@ EOF
+  ['sabayon']="dev-python/psycopg:2"
+  ['rhel']="python-psycopg2"
+  ['suse']="python-psycopg2"
+  ['default']="python-psycopg2"
+)
 
 declare -A pkg_python3_psycopg2=(
-	 ['alpine']="py3-psycopg2"
-	   ['arch']="python-psycopg2"
-	 ['centos']="WARNING|"
-	 ['debian']="WARNING|"
-	 ['gentoo']="dev-python/psycopg"
-	['sabayon']="dev-python/psycopg:2"
-	   ['rhel']="WARNING|"
-	   ['suse']="WARNING|"
-	['default']="WARNING|"
-	)
+  ['alpine']="py3-psycopg2"
+  ['arch']="python-psycopg2"
+  ['centos']="WARNING|"
+  ['debian']="WARNING|"
+  ['gentoo']="dev-python/psycopg"
+  ['sabayon']="dev-python/psycopg:2"
+  ['rhel']="WARNING|"
+  ['suse']="WARNING|"
+  ['default']="WARNING|"
+)
 
 declare -A pkg_python_pip=(
-	 ['alpine']="py-pip"
-	 ['gentoo']="dev-python/pip"
-	['sabayon']="dev-python/pip"
-	['default']="python-pip"
-	)
+  ['alpine']="py-pip"
+  ['gentoo']="dev-python/pip"
+  ['sabayon']="dev-python/pip"
+  ['default']="python-pip"
+)
 
 declare -A pkg_python3_pip=(
-	 ['alpine']="py3-pip"
-	   ['arch']="python-pip"
-	 ['centos']="WARNING|"
-	 ['gentoo']="dev-python/pip"
-	['sabayon']="dev-python/pip"
-	   ['rhel']="WARNING|"
-	['default']="python3-pip"
-	)
+  ['alpine']="py3-pip"
+  ['arch']="python-pip"
+  ['centos']="WARNING|"
+  ['gentoo']="dev-python/pip"
+  ['sabayon']="dev-python/pip"
+  ['rhel']="WARNING|"
+  ['default']="python3-pip"
+)
 
 declare -A pkg_python_pymongo=(
-	 ['alpine']="WARNING|"
-	   ['arch']="python2-pymongo"
-	 ['centos']="WARNING|"
-	 ['debian']="python-pymongo"
-	 ['gentoo']="dev-python/pymongo"
-	   ['suse']="python-pymongo"
-	['default']="python-pymongo"
-	)
+  ['alpine']="WARNING|"
+  ['arch']="python2-pymongo"
+  ['centos']="WARNING|"
+  ['debian']="python-pymongo"
+  ['gentoo']="dev-python/pymongo"
+  ['suse']="python-pymongo"
+  ['default']="python-pymongo"
+)
 
 declare -A pkg_python3_pymongo=(
-	 ['alpine']="WARNING|"
-	   ['arch']="python-pymongo"
-	 ['centos']="WARNING|"
-	 ['debian']="python3-pymongo"
-	 ['gentoo']="dev-python/pymongo"
-	   ['suse']="python3-pymongo"
-	['default']="python3-pymongo"
-	)
+  ['alpine']="WARNING|"
+  ['arch']="python-pymongo"
+  ['centos']="WARNING|"
+  ['debian']="python3-pymongo"
+  ['gentoo']="dev-python/pymongo"
+  ['suse']="python3-pymongo"
+  ['default']="python3-pymongo"
+)
 
 declare -A pkg_python_requests=(
-	 ['alpine']="py-requests"
-	   ['arch']="python2-requests"
-	 ['centos']="python-requests"
-	 ['debian']="python-requests"
-	 ['gentoo']="dev-python/requests"
-	['sabayon']="dev-python/requests"
-	   ['rhel']="python-requests"
-	   ['suse']="python-requests"
-	['default']="python-requests"
-
-   ['alpine-3.1.4']="WARNING|"
-   ['alpine-3.2.3']="WARNING|"
-	)
+  ['alpine']="py-requests"
+  ['arch']="python2-requests"
+  ['centos']="python-requests"
+  ['debian']="python-requests"
+  ['gentoo']="dev-python/requests"
+  ['sabayon']="dev-python/requests"
+  ['rhel']="python-requests"
+  ['suse']="python-requests"
+  ['default']="python-requests"
+  [\
```

Comment:
This doesn't look right.

Reply:
Good catch! Uncertain how this happened; but fixed.